src/main/java/org/codeforamerica/messaging/models/SmsMessage.java

```diff
@@ -27,14 +26,12 @@ public class SmsMessage {
     @Id
     @GeneratedValue(strategy= GenerationType.IDENTITY)
     private Long id;
-    @NotBlank
```

Comment:
I see the `@NotBlank` validators for `toPhone` and `fromPhone` were removed, did you consider adding `@NotNull` (or whatever is applicable)?